apps/builder/app/builder/features/style-panel/style-source/style-source-control.tsx

```diff
@@ -318,14 +324,15 @@ export const StyleSourceControl = ({
         disabled={disabled}
         aria-current={selected && state === undefined}
         role="button"
-        onClick={onSelect}
         hasError={error !== undefined}
       >
         <Flex grow css={{ padding: theme.spacing[2] }}>
```

Comment:
this padding can be put on flex inside of button

Reply:
no bleeding necessary

Reply:
can't, check token name editing, it will loose the "border", so you won't see the token

Reply:
could add the border back during editing on the container but not sure this is any better